lib/docker/image.rb

```diff
@@ -227,7 +227,16 @@ def search(query = {}, connection = Docker.connection, creds = nil)
     # Import an Image from the output of Docker::Container#export. The first
     # argument may either be a File or URI.
     def import(imp, opts = {}, conn = Docker.connection)
-      open(imp) do |io|
+      require 'open-uri'
```

Comment:
I don't frequently open IO objects beside local files, what's the benefit here vs the prior `open` method?

Reply:
Seamless Ruby 2.7 compatibility.